build.py

```diff
@@ -1,105 +1,60 @@
+      elif line.startswith('Last modified:'):
         header_lines[i] = line.replace('${current_time}', current_time)
       elif line.startswith('Entries:'):
         header_lines[i] = line.replace('${num_entries}', str(num_entries))
       elif line.startswith('Format:'):
         header_lines[i] = line.replace('${format}', format)
+
       # Add comment symbols before every header line
       if format == 'ublock' or format == 'abp':
         header_lines[i] = ('! ' + header_lines[i])
       if format == 'hosts' or format == 'domains' or format == 'wildcard_domains' or format == 'wildcard_urls':
         header_lines[i] = ('# ' + header_lines[i])
 
-    # Add proper filterlist formatting to the domains
-    domains_lines = f_domains.readlines()
-    for i, line in enumerate(domains_lines):
-      if not line.startswith('!'):
-        if format == 'ublock' or format == 'abp':
-          domains_lines[i] = line.replace("\n", '')
-          domains_lines[i] = ('||' + domains_lines[i] + '^' + "\n")
-        elif format == 'hosts':
-          domains_lines[i] = ('0.0.0.0 ' + domains_lines[i])
-        elif format == 'wildcard_domains':
-          domains_lines[i] = ('*.' + domains_lines[i])
-        elif format == 'wildcard_urls':
-          domains_lines[i] = line.replace("\n", '')
-          domains_lines[i] = ('*://*.' + domains_lines[i] + '/*' + "\n")
-      if line.startswith('!'):
-        if format == 'hosts' or format == 'domains' or format == 'wildcard_domains' or format == 'wildcard_urls':
-          domains_lines[i] = line.replace('!', '#')
-      line.format()
-
-    # Add proper filterlist formatting to the plus version extra domains
-    if plus_version == True:
-      with open(sitelist_file_plus, 'r') as f_domains_plus:
-        domains_lines_plus = f_domains_plus.readlines()
-        for i, line in enumerate(domains_lines_plus):
-          if not line.startswith('!'):
-            if format == 'ublock' or format == 'abp':
-              domains_lines_plus[i] = line.replace("\n", '')
-              domains_lines_plus[i] = ('||' + domains_lines_plus[i] + '^' + "\n")
-            elif format == 'hosts':
-              domains_lines_plus[i] = ('0.0.0.0 ' + domains_lines_plus[i])
-            elif format == 'wildcard_domains':
-              domains_lines_plus[i] = ('*.' + domains_lines_plus[i])
-            elif format == 'wildcard_urls':
-              domains_lines_plus[i] = line.replace("\n", '')
-              domains_lines_plus[i] = ('*://*.' + domains_lines_plus[i] + '/*' + "\n")
-          if line.startswith('!'):
-            if format == 'hosts' or format == 'domains' or format == 'wildcard_domains' or format == 'wildcard_urls':
-             domains_lines_plus[i] = line.replace('!', '#')
-          line.format()
+    # Add proper filterlist formatting
+    domains_lines = formatFile(sitelist_file, format)
+    if(is_sitelist_plus):
```

Comment:
Another space suggestion...
```suggestion
    if (is_sitelist_plus):
```